src/Docker.DotNet/Microsoft.Net.Http.Client/ManagedHandler.cs

```diff
@@ -156,7 +156,7 @@ private bool IsAllowedRedirectResponse(HttpRequestMessage request, HttpResponseM
                 if (_socketOpener != null)
```

Comment:
Its possible its just my state of tired but what are the changes in this file? They look like space only?